dubbo-admin-server/src/main/java/org/apache/dubbo/admin/config/SwaggerConfiguration.java

```diff
@@ -29,6 +30,7 @@
 
 @Configuration
 @EnableSwagger2
+@ConditionalOnProperty(name = "swagger.enable", havingValue = "true")
```

Comment:
Should add `matchIfMissing = true` default enabled.

Reply:
ok, added it